File: Leetcode/217-Contains-Duplicate/ContainsDuplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        if not nums or len(nums)<2:
            return False
        nums.sort();
        i=0
        size = len(nums)
        while i<size-1:
            if nums[i] == nums[i+1]:
                return True
            i+=1
        return False
    

File: Leetcode/217-Contains-Duplicate/ContainsDuplicate.go
import "sort"

func containsDuplicate(nums []int) bool {
	if nums == nil || len(nums) < 2 {
		return false
	}
	sort.Ints(nums)
	size := len(nums)
	for ind, val := range nums {
		if ind < size-1 && val == nums[ind+1] {
			return true
		}
	}
	return false
}
